src/test/api/karate/cards/postCardWithStoreOnlyInArchives.feature

```diff
@@ -0,0 +1,126 @@
+Feature: postCardWithStoreOnlyInArchives
+
+  Background:
+
+    * def signIn = callonce read('../common/getToken.feature') { username: 'operator1_fr'}
+    * def authToken = signIn.authToken
+    * def signInAdmin = callonce read('../common/getToken.feature') { username: 'admin'}
+    * def authTokenAdmin = signInAdmin.authToken
+    * def perimeter =
+"""
+{
+  "id" : "perimeter",
+  "process" : "api_test",
+  "stateRights" : [
+      {
+        "state" : "messageState",
+        "right" : "ReceiveAndWrite"
+      }
+    ]
+}
+"""
+
+    * def perimeterArray =
+"""
+[   "perimeter"
+]
+"""
+
+  Scenario: postCardWithout StoreOnlyInArchives
+
+    * def cardWithoutStoreOnlyInArchives =
+"""
+{
+	"publisher" : "operator1_fr",
+	"processVersion" : "1",
+	"process"  :"api_test",
+	"processInstanceId" : "process_postCardWithoutStoreOnlyInArchives",
+	"state": "messageState",
+	"groupRecipients": ["Dispatcher"],
+	"severity" : "ALARM",
+	"startDate" : 1583943924000,
+	"endDate" : 1584943924000,
+	"summary" : {"key" : "defaultProcess.summary"},
+	"title" : {"key" : "defaultProcess.title"},
+	"data" : {"message":"a message without STORE_ONLY_IN_ARCHIVES action"}
+}
```

Comment:
I think you can only keep the test of the card with the action STORE_ONLY_IN_ARCHIVES as the case where the action is not set is already tested in other karate tests